boltdesignsystem/bolt
# DS-629 Remove Code Snippet WC usage in PL

## Jira

https://pegadigitalit.atlassian.net/browse/DS-629

## Summary

The Old `<bolt-code-snippet>` component was replaced with a `<code>` markup for inline snippets, attributes were disregarded in these examples. Other block snippets instances were replaced with the new Code Snippet Twig component. 

## Details

The codebase was searched for `<bolt-code-snippet`.  All these instances were replaced with the new code snippet.

## How to test

Pull the branch. Check if there are no more instances of the old code component snippet - `<bolt-code-snippet`. Make sure the new code snippets render properly.
